locale/de/locale.cfg

```diff
@@ -0,0 +1,31 @@
+[item-name]
+bni_battery-pack=Geladenes Akku Pack
+bni_battery-pack-2=Geladenes Hochleistungs Akku Pack
+bni_battery-pack-empty=Leeres Akku Pack
+bni_battery-pack-2-empty=Leeres Hochleistungs Akku Pack
+
+
+[entity-name]
+bni_electric-locomotive=Elektrische Lock
+bni_charging-station=Akku Ladestation
+bni_rapid-charging-station=Akku Schnell Ladestation
+
+
+[entity-description]
+bni_electric-locomotive=Fährt nach Plan und zieht Güterwagongs. Einsteigen mit __CONTROL__toggle-driving__, Zug menu öffnen mit __CONTROL__build__. Mit Akku Packs angetrieben, können zum Aufladen getauscht werden.
```

Comment:
I would change the last sentence to: Wird mit Akkupacks aufgeladen und können zum Laden getauscht werden.